config/auth_gcp_google_id.go

```diff
@@ -44,7 +45,8 @@ func (c GoogleDefaultCredentials) Configure(ctx context.Context, cfg *Config) (f
 		return nil, err
 	}
 	logger.Infof(ctx, "Using Google Default Application Credentials for Accounts API")
-	return serviceToServiceVisitor(inner, platform, "X-Databricks-GCP-SA-Access-Token"), nil
+	visitor := serviceToServiceVisitor(inner, platform, "X-Databricks-GCP-SA-Access-Token")
+	return credentials.NewCredentialsProvider(visitor), nil
```

Comment:
Can we not use inner.Token here for OAuth?

Reply:
I think I just missed that one. Updated.